packages/esm-patient-search-app/src/patient-search-page/patient-search-lg.component.tsx

```diff
@@ -48,17 +50,20 @@ const PatientSearchComponent: React.FC<PatientSearchComponentProps> = ({
   const handlePatientSelection = useCallback(
     (evt, patientUuid: string) => {
       evt.preventDefault();
-      if (onPatientSelect) {
-        onPatientSelect(patientUuid);
+      if (selectPatientAction) {
+        selectPatientAction(patientUuid);
       } else {
         navigate({
-          to: interpolateString(config.search.patientResultUrl, {
+          to: `${interpolateString(config.search.patientResultUrl, {
             patientUuid: patientUuid,
-          }),
+          })}/${encodeURIComponent('Patient Summary')}`,
         });
       }
+      if (hidePanel) {
+        hidePanel();
+      }
```

Comment:
I would move this before the `if ... else` since there is a chance that the `navigate` will unmount the component before you get a chance to hide the panel.... or do I have this backwards?

Reply:
Sure!
Nice catch!

Reply:
On click, it will run the whole function, and even, hidePanel will be faster to unmount the component, than the `navigate`. The process for `navigate` is to first change the url and then mount the new component, but `hidePanel` will directly unmount the search component.